File: POLIMORFISMO/EJER2/JAVA/Mesero.java
public class Mesero {
    private String nombre;
    private int sueldoMes;
    private int horasExtra;
    private float sueldoHora;
    private float propina;

    public Mesero(String nombre, int sueldoMes, int horasExtra, float sueldoHora, float propina) {
        this.nombre = nombre;
        this.sueldoMes = sueldoMes;
        this.horasExtra = horasExtra;
        this.sueldoHora = sueldoHora;
        this.propina = propina;
    }

    public float sueldoTotal() {
        return sueldoMes + horasExtra * sueldoHora + propina;
    }

    public void mostrarSiSueldoEs(int x) {
        if (sueldoMes == x) {
            System.out.println("Mesero: " + nombre);
        }
    }
}

File: POLIMORFISMO/EJER2/Python/Mesero.py
class Mesero:
    def __init__(self, nombre, sueldo_mes, horas_extra, sueldo_hora, propina):
        self.nombre = nombre
        self.sueldo_mes = sueldo_mes
        self.horas_extra = horas_extra
        self.sueldo_hora = sueldo_hora
        self.propina = propina

    def sueldo_total(self):
        return self.sueldo_mes + self.horas_extra * self.sueldo_hora + self.propina

    def mostrar_si_sueldo_es(self, x):
        if self.sueldo_mes == x:
            print(f"Mesero: {self.nombre}")


